news/rustconf-2024-what-to-expect.md

```diff
@@ -0,0 +1,85 @@
+
+**Workshop 3: Unleash Your Computer’s Power with Rust - Safely**
+
+*Instructor: Herbert Wolverson*
+
+The workshop aims to show the value of Rust despite its complexity. Participants will learn to efficiently distribute workloads, communicate asynchronously, and utilize their computer’s capabilities using Rust’s safety features. The session will also explain Rust’s unique system safety and error prevention approach.
+
+Interested in registering for a workshop? You can add one to your existing order or register for the first time with a workshop [here](https://www.eventbrite.com/e/rustconf-2024-tickets-865842106047). Just select “Get tickets”. Workshops cost $250.
+
+### Virtual Experience
+
+If you can’t attend in person this year, you can still join for the free virtual RustConf. Once you[<u>register</u>](https://rustconf.com/register/) for RustConf, we’ll send you a link to join our dedicated Discord server a week before the event. On Discord, you’ll find information about livestreams, opportunities for Q&A, and the chance to interact with other attendees and speakers. The virtual platform will not only be an essential tool for those joining us remotely, but also for those onsite interested in asking our speakers questions, capturing their experience, and making lasting connections with other attendees and sponsors.
+
+### Rust Global:
+
+Rust Global is a Rust Foundation-hosted half-day event taking place on Friday, September 13th. The event will feature talks and discussions about the use of Rust in enterprise, government, and global leadership settings. It will be free for anyone with a Nonprofit or Industry/Government ticket (while capacity remains). Learn more on the[<u>registration page</u>](https://www.eventbrite.com/e/rustconf-2024-tickets-865842106047?aff=oddtdtcreator).
+
+We are thrilled to share that Rust Foundation security Engineer Walter Pearce has just been announced as the final Rust Global speaker! Learn more about Walter’s talk and the complete lineup of Rust Global sessions here: [https://rustconf.com/programs/#rust\_global](https://rustconf.com/programs/#rust_global)
+
+### RustConf “UnConference”:
+
+On Friday, September 13th, from 9am to 3pm, RustConf will feature the much-loved, freeform “UnConf” event, which is dedicated to open dialogue about the Rust Project after the main programming concludes. The UnConference is free and open to anyone with a RustConf ticket.
+
+Full Schedule:[<u>https://rustconf.com/schedule/</u>](https://rustconf.com/schedule/)
+
+Register:[<u>https://rustconf.com/register/</u>](https://rustconf.com/register/)
+
+---
+
+### See You in September!
+
+Whether you are attending RustConf in Montreal or remotely from wherever you live, the Rust Foundation + RustConf team can’t wait to host you at this annual gathering of the Rustacean community 🦀
+
+[https://rustconf.com](https://rustconf.com)
+
+&nbsp;
+
```

Comment:
Add: 
## Low Ticket Warning
RustConf tickets are going fast! If you plan to join us in Montreal, be sure to grab your spot before they're gone!. Register [here](https://www.eventbrite.com/e/rustconf-2024-tickets-865842106047?aff=oddtdtcreator).